graphql/server.go

```diff
@@ -486,6 +503,7 @@ func CreateJSONSocket(ctx context.Context, socket JSONSocket, schema *Schema, ma
 	}
 }
 
+// CreateJSONSocketWithMutationSchema is deprecated. Consider using CreateConnection instead.
 func CreateJSONSocketWithMutationSchema(ctx context.Context, socket JSONSocket, schema, mutationSchema *Schema, makeCtx MakeCtxFunc, logger GraphqlLogger) *conn {
 	return &conn{
```

Comment:
can we make these call CreateConnection instead?

Reply:
done